### PR TITLE
Fix minikube profile in e2e tests

### PR DESCRIPTION
**Pull request by dgrisonnet**
This PR is a follow-up to the following discussion: https://github.com/kubernetes/kube-state-metrics/pull/1119#discussion_r408271195

cc @brancz

### CODE REVIEW

File: tests/e2e.sh [hunk 1]
```diff
@@ -67,13 +67,14 @@ touch "${HOME}"/.kube/config
 
 export KUBECONFIG=$HOME/.kube/config
 
-if [[ "$MINIKUBE_DRIVER" != "none" ]]; then 
-  export MINIKUBE_PROFILE_ARG=" --profile ${MINIKUBE_PROFILE}"
+if [[ "$MINIKUBE_DRIVER" != "none" ]]; then
```
Comment (tariq1890) [inline tests/e2e.sh:70]: Let's set the default value to the variable first. This eliminates the need for the else statement.

Once the setting is done, we can then `export` the var as opposed to `export`ing in every conditional path

**Comment by tariq1890** [comment]
Thanks for the PR @dgrisonnet :)

**Comment by dgrisonnet** [comment]
@olivierlemasle good catch, I completely forgot about that. It should be fixed now :).

**Comment by brancz** [comment]
Thanks a lot for the quick follow up! lgtm :tada: 

Also congrats on your first contribution to the Kubernetes project! :tada: :partying_face: 

/lgtm
/approve

**Pull request merged**